components/organisms/CreateLessonWithinCourseAdminPortal/CreateLessonWithinCourseAdminPortal.tsx

```diff
@@ -175,7 +192,7 @@ export const CreateLessonWithinCourseAdminPortal = () => {
           <Button
             type="submit"
             className={styles["button__submit"]}
-            disabled={!selectedResource}
+            // disabled={!selectedResource}
```

Comment:
do we need this commented line?

Reply:
That line is not necessary in the code, it was removed.